src/Service/ZaakService.php

```diff
@@ -191,6 +191,59 @@ private function checkZaakType(array $case)
     }//end checkZaakType()
 
 
+    /**
+     * Gets the actual document from a different endpoint that has the metadata.
+     *
+     * @param string $documentNumber document number (not the id).
+     *
+     * @return array
+     */
+    private function getActualDocument(string $documentNumber): array
+    {
+        try {
+            isset($this->style) === true && $this->style->info("Fetching actual document: $documentNumber..");
```

Comment:
(Do we ever reach this code using a command? If not remove $this->style)

Reply:
we do :)